plugins/woocommerce/src/Blocks/BlockTypes/ProductFiltersOverlay.php

```diff
@@ -30,10 +33,49 @@ protected function get_block_type_style() {
 	 * @return string Rendered block type output.
 	 */
 	protected function render( $attributes, $content, $block ) {
-		ob_start();
-		printf( '<div>%s</div>', esc_html__( 'Filters Overlay', 'woocommerce' ) );
-		$html = ob_get_clean();
+		return $content;
+	}
+
+	/**
+	 * Extra data passed through from server to client for block.
+	 *
+	 * @param array $attributes  Any attributes that currently are available from the block.
+	 *                           Note, this will be empty in the editor context when the block is
+	 *                           not in the post content on editor load.
+	 */
+	protected function enqueue_data( array $attributes = [] ) {
+		parent::enqueue_data( $attributes );
+
+		$template_part_edit_uri = '';
+
+		if (
+			current_user_can( 'edit_theme_options' ) &&
+			( wc_current_theme_is_fse_theme() || current_theme_supports( 'block-template-parts' ) )
+		) {
+			$theme_slug = BlockTemplateUtils::theme_has_template_part( 'product-filters-overlay' ) ? wp_get_theme()->get_stylesheet() : BlockTemplateUtils::PLUGIN_SLUG;
+
+			$site_editor_uri = add_query_arg(
+				array(
+					'canvas' => 'edit',
+					'path'   => '/template-parts/single',
```

Comment:
I got redirected to this URL after clicking on the edit link: `http://woo.test/wp-admin/site-editor.php?canvas=edit&postId=woocommerce/woocommerce//product-filters-overlay&postType=wp_template_part`. I'm using WP 6.4 which is the minimum supported WP version.

Reply:
Yes that should be correct. It takes you to the overlay template part. I got the same in both WP 6.4 and 6.5.